lib/bandit/http1/handler.ex

```diff
@@ -197,4 +197,31 @@ defmodule Bandit.HTTP1.Handler do
 
   def handle_info({:EXIT, _pid, :normal}, {socket, state}),
     do: {:noreply, {socket, state}, socket.read_timeout}
+
+  def handle_info(msg, state) do
+    # Copied verbatim from lib/elixir/lib/gen_server.ex
```

Comment:
Ideally we'd just call `super(msg, state)` here but calling super on (most) GenServer callbacks was deprecated a [while ago](https://github.com/elixir-lang/elixir/commit/12a4ea4c9fb83dd9f370fc9247207c02cf0d5443#diff-07e9c029fa058bfd89482ef285ba58c16a3f3a64158a3267d8dea0391710da26)